vmm/src/vstate.rs

```diff
@@ -278,6 +281,41 @@ impl Vcpu {
         Ok(())
     }
 
+    #[cfg(target_arch = "aarch64")]
+    /// Configures an aarch64 specific vcpu and should be called once per vcpu from the vcpu's thread.
```

Comment:
nit: it is not actually necessary to call this from the vcpu's thread, and we actually don't do that.